tests/hpc/product_migration.pm

```diff
@@ -8,7 +8,7 @@
 # without any warranty.
 
 # Summary: Checking ability to migrate from SLE 12 with HPC module to SLE 12 HPC Product
-# Maintainer: Anton Smorodskyi <asmorodskyi@suse.com>
+# Maintainer: N/A
```

Comment:
same here , not sure what should I write :thinking:

Reply:
@foursixnine  it is SLE for HPC , so it is applicable , right ?

Reply:
No :) HPC is a separate thing :D